debian/context/install.sh

```diff
@@ -86,11 +86,6 @@ then
     grub-install --target=x86_64-efi --efi-directory=${EFI_MOUNTPOINT} --boot-directory=/boot --bootloader-id=${BOOTLOADER_ID}
     update-grub2
     dpkg-reconfigure grub-efi-amd64-bin
-else
-    echo "System was booted with Bios"
-    grub-install --boot-directory=/boot --bootloader-id=${BOOTLOADER_ID}
-    update-grub2
-    dpkg-reconfigure grub
```

Comment:
this code is actually not executed in physical partitions and causes trouble with Non-EFI VMs as with our test setup. also the invocation of grub-install is completely wrong in this case.

Reply:
But we should at least fail if booted in bios mode

Reply:
grub-installation was moved to the end of the install script and the script fails in this case